Client/Assets/Images.xcassets/Shopping/shoppingNoAnalysisImage.imageset/Contents.json

```diff
@@ -1,7 +1,17 @@
 {
   "images" : [
     {
-      "filename" : "shoppingNoAnalysisImage.pdf",
+      "filename" : "mobile iOS no-analysis 1x 280x108 light 72dpi MARGIN.png",
+      "idiom" : "universal"
```

Comment:
I know it was mentioned the image is by design pixelated but I see the naming contains 1x. Shouldn't we have 2 and 3x. variants as well?

Reply:
I showed design the screenshots with the 1x images and they approved the usage like this.